src/Operations.jl

```diff
@@ -2276,7 +2276,7 @@ function test(ctx::Context, pkgs::Vector{PackageSpec};
             printpkgstyle(ctx.io, :Testing, "Running tests...")
             flush(ctx.io)
             code = gen_test_code(source_path; test_args)
-            cmd = `$(Base.julia_cmd()) $(flags) --eval $code`
+            cmd = Cmd(`$(Base.julia_cmd()) $(flags) --eval $code`; env = julia_args.env)
```

Comment:
What about the other parts of the `julia_args` `Cmd`?

Is it safe to ignore `julia_args.cpus`, `julia_args.dir`, `julia_args.flags` (!= `flags`), and `julia_args.ignorestatus`?